content/en/docs/concepts/services-networking/service.md

```diff
@@ -241,9 +241,18 @@ There are a few reasons for using proxying for Services:
   on the DNS records could impose a high load on DNS that then becomes
   difficult to manage.
 
+Below we'll describe how various kube-proxy implementations work.  In general
+you should note that, when running `kube-proxy`, kernel level rules may be
+modified (for example, iptables rules might get created), which won't get cleaned up, 
+in some cases until you reboot.  Thus, running kube-proxy is something that should
+only be done by an administrator which understands the consequences of having a
+low leve, priveliged network proxying service on a machine.  Although the `kube-proxy`
```

Comment:
```suggestion
low level, privileged network proxying service on a computer.  Although the `kube-proxy`
```

Reply:
Maybe don't mention the unofficial `cleanup` subcommand at all (if it's not supported).